.github/workflows/olm_tests.yaml

```diff
@@ -41,20 +41,15 @@ jobs:
       run: |
         # TODO: Use operator-sdk run bundle and bundle-upgrade commands for
         # testing with OLM.
-        REGISTRY_PORT=5000
-        REPO=localhost:${REGISTRY_PORT}
-        IMG=${REPO}/gatekeeper-operator:${GITHUB_SHA}
-        PREV_BUNDLE_INDEX_IMG=quay.io/gatekeeper/gatekeeper-operator-bundle-index:v$(cat REPLACES_VERSION)
```

Comment:
(The `PREV_BUNDLE_INDEX_IMG` was removed because these are the default values in the Makefile)